consensus/blake3pow/difficulty.go

```diff
@@ -18,7 +18,7 @@ const (
 )
 
 // MakeDifficultyCalculatorU256 creates a difficultyCalculator with the given bomb-delay.
-// the difficulty is calculated with Byzantium rules, which differs in
```

Comment:
I lol'd at this. might just delete the comment entirely, as it no longer makes sense.

Reply:
Fixed